.npmrc

```diff
@@ -1,4 +1,5 @@
 shamefully-hoist=true
 strict-peer-dependencies=false
 shell-emulator=true
-package-manager-strict=false
\ No newline at end of file
+package-manager-strict=false
+https-proxy=socks://127.0.0.1:1080
```

Comment:
??
Perhaps this file should not be modified

Reply:
...my bad